### DISCUSSION TITLE
Multi datacenter alerting

### DISCUSSION THREAD

**Discussion by msartor92** [grafana/grafana#69778] [Alerting]
Hi all,

CONTEXT
---------------
I'm using Grafana 9.5.1 in two Datacenter and MariaDB as Grafana database with cross DC replication. This work well because I had users, dashboard and so on replicated so I do not need to configure them in both DC.

One of my DC is for disaster recovery purpose and many application is not running so I've not data to collect, in that case i won't receive alert. But I won't disable alerting feature because in other cases I need alerting.

To configure this I think to use Silence and Tag, more in detail if I configure a silence interval for alert with tag environment = dr I will not receive notification from that DC.

PROBLEM
---------------
What I can't understand how to setup an alter with a tag value initialized from something like an environment variables or other external tool so I have different values in each Datacenter and values are not replicated from database 

Could someone help me?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
